examples/time-zone/time_zone.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/io;
+import ballerina/time;
+
+public function main() returns error? {
+    // Event recorded in UTC (2022-01-29 22:48:00)
+    time:Utc eventUtcTime = check time:utcFromString("2022-01-29T22:48:00Z");
+    io:println("Event Time in UTC: ", eventUtcTime);
+
+    // Load the system's default time zone
```

Comment:
All comments should end with a period.
```suggestion
    // Load the system's default time zone.
```

Reply:
Resolved via [ba00cf8](https://github.com/ballerina-platform/ballerina-distribution/pull/5684/commits/ba00cf8eba4026556ebb79f2b54df3656d15edd5)